lbz/dev/test.py

```diff
@@ -11,25 +15,70 @@ class Client:
     Client created for testing purposes.
     """
 
-    def __init__(self, resource):
+    def __init__(
+        self, resource: Type[Resource]
+    ):  # here, you are expecting to get a class, not its instance
```

Comment:
I put this comment in the proposed snippet of code only to simplify understanding, please, remove it from the code 😉